### PR TITLE
Fix rdsr normalizer crashes for events with no filter

### PR DESCRIPTION
**Pull request by BwKodex**
Adds a fix for the RDSR normalizer crashing on normalizing the X-Ray filter material thickness for events that have no filter. See bug report #37.

Also cleans up the RDSR parser/normalizer test code a bit by moving them into separate integration test files

### CODE REVIEW

File: tests/integrationtests/conftest.py [hunk 1]
```diff
@@ -0,0 +1,52 @@
+from pathlib import Path
+
+import pandas as pd
+import pydicom
+import pytest
+
+from pyskindose.rdsr_parser import rdsr_parser
+
+
+@pytest.fixture(scope="function")
```
Comment (MaxHellstrom) [inline tests/integrationtests/conftest.py:10]: I need to read up on fixtures =)

**Review by MaxHellstrom — APPROVED** [review]
have a look at the pytest warnings in test_rdsr_normalizer

File: tests/integrationtests/conftest.py (hunk 1 above)
Comment (BwKodex) [reply tests/integrationtests/conftest.py:10]: Yes you do 😜

File: tests/integrationtests/test_rdsr_normalizer.py [hunk 2]
```diff
@@ -0,0 +1,101 @@
...
+        data_norm_philips[KEY_NORMALIZATION_FILTER_SIZE_COPPER][0],
+        data_norm_siemens[KEY_NORMALIZATION_FILTER_SIZE_ALUMINUM][0],
+        data_norm_siemens[KEY_NORMALIZATION_FILTER_SIZE_COPPER][0],
+    ]
+
+    # Assert
+    assert actual == expected
+
+
+def test_xray_filter_mean_thickness_calculated_for_each_filter_material_separately(allura_parsed):
```
[the remark this answers is not in the document]
Comment (BwKodex) [reply tests/integrationtests/test_rdsr_normalizer.py:36]: Isn't that also present when the normalizer is run normally?

**Pull request merged**